### PR TITLE
adding parsing test

### PR DESCRIPTION
**Pull request by Jimexist**
adding parsing test in antlr parsing

### CODE REVIEW

**Comment by dminkovsky** [comment]
Thank you for this PR. Out of curiosity, does this address a particular issue you were encountering?

**Comment by Jimexist** [comment]
it... sort of. i was seeing failure for parsing unicode. in fact there was no implementation yet so i went on to add minimal test for non-unicode cases. would be happy to do another pr for unicode support though.

File: src/main/java/graphql/parser/GraphqlAntlrToLanguage.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 public class GraphqlAntlrToLanguage extends GraphqlBaseVisitor<Void> {
 
-    Document result;
+    private Document result;
```
Comment (andimarek) [inline src/main/java/graphql/parser/GraphqlAntlrToLanguage.java]: Could you revert it? This change is unrelated to the parsing.

File: src/test/groovy/graphql/parser/GraphqlAntlrToLanguageTest.groovy [hunk 2]
```diff
@@ -0,0 +1,66 @@
+package graphql.parser
+
+import spock.lang.Ignore
+import spock.lang.Specification
+/**
```
Comment (andimarek) [inline src/test/groovy/graphql/parser/GraphqlAntlrToLanguageTest.groovy]: Could you remove this comment? We don't maintain author comments.

File: src/main/java/graphql/parser/GraphqlAntlrToLanguage.java [hunk 3]
```diff
@@ -688,9 +691,12 @@ private Value getValue(GraphqlParser.ValueContext ctx) {
         throw new ShouldNotHappenException();
     }
 
-    private String parseString(String string) {
+    static String parseString(String string) {
+        assert string.length() >= 2;
```
Comment (andimarek) [inline src/main/java/graphql/parser/GraphqlAntlrToLanguage.java]: We don't use assert statements in prod code: they are turned off by default and don't have any effect.

Also it is not really needed to ensure the length or that the string starts wit `"`, this is already ensured by the parser/lexer.

**Comment by bbakerman** [comment]
re implemented here : https://github.com/graphql-java/graphql-java/pull/464

**Comment by Jimexist** [comment]
@bbakerman been busy recently, thanks for taking over
